boilerplate/app/components/Toggle/Toggle.tsx

```diff
@@ -118,7 +118,7 @@ export interface BaseToggleInputProps<T> {
 /**
  * Renders a boolean input.
  * This is a controlled component that requires an onValueChange callback that updates the value prop in order for the component to reflect user actions. If the value prop is not updated, the component will continue to render the supplied value prop instead of the expected result of any user actions.
- * @see [Documentation and Examples]{@link https://docs.infinite.red/ignite-cli/boilerplate/components/Toggle/}
+ * @see [Documentation and Examples]{@link https://docs.infinite.red/ignite-cli/boilerplate/app/components/Toggle/}
```

Comment:
Actually this is not a doc page, we'd need to move this comment into the Checkbox, Radio and Switch components that fall under the Toggle directory

Reply:
After double-checking, I realized that the `Toggle` component doesn't have an example page, so I removed this line. I did, however, add links to the `Checkbox`, `Radio`, and `Switch` components.